### PR TITLE
chore(test): Add images to karma, fix warnings

### PR DESCRIPTION
**Pull request by k88hudson**
This PR

*  fixes unwanted React warnings
* overrides `console.error` to throw in tests if unwanted errors are logged, and provides a method of temporarily disabling this 
* adds test images to karma so 404s wont happen

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/activity-streams/461)

### CODE REVIEW

File: content-test/index.js [hunk 1]
```diff
@@ -1,10 +1,17 @@
 const req = require.context(".", true, /\.test.js$/);
 const files = req.keys();
+const {overrideConsoleError} = require("test/test-utils");
 
-const chai = require("chai");
```
Comment (k88hudson) [inline content-test/index.js]: this is unused

**Comment by k88hudson** [comment]
R? @pdehaan or @rlr?

**Comment by rlr** [comment]
travis says:

```
/home/travis/build/mozilla/activity-streams/content-test/index.js

  9:31  error  Missing semicolon  semi
```

Otherwise, things LGTM!

**Comment by pdehaan** [comment]
@k88hudson I'm still seeing oodles of these in the Travis logs: https://travis-ci.org/mozilla/activity-streams#L1804

```
    GroupedActivityFeed
      valid grouped activity feed
        ✔ should create the element
        ✔ should render an ActivityFeed for each date
        ✔ shouldn't render title if there are no sites
08 04 2016 15:34:08.443:WARN [web-server]: 404: /base/data/content/favicons/images/nytimes-com.png
08 04 2016 15:34:08.449:WARN [web-server]: 404: /base/data/content/favicons/images/salesforce-com.png
08 04 2016 15:34:08.450:WARN [web-server]: 404: /base/data/content/favicons/images/office-com.png
08 04 2016 15:34:08.450:WARN [web-server]: 404: /base/data/content/favicons/images/foxnews-com.png
08 04 2016 15:34:08.457:WARN [web-server]: 404: /base/data/content/favicons/images/about-com.png
08 04 2016 15:34:08.458:WARN [web-server]: 404: /base/data/content/favicons/images/linkedin-com.png
08 04 2016 15:34:08.468:WARN [web-server]: 404: /base/data/content/favicons/images/americanexpress-com.png
08 04 2016 15:34:08.472:WARN [web-server]: 404: /base/data/content/favicons/images/yelp-com.png
08 04 2016 15:34:08.472:WARN [web-server]: 404: /base/data/content/favicons/images/capitalone-com.png
08 04 2016 15:34:08.473:WARN [web-server]: 404: /base/data/content/favicons/images/homedepot-com.png
08 04 2016 15:34:08.479:WARN [web-server]: 404: /base/data/content/favicons/images/twitch-tv.png
```

**Pull request merged**